operators/go/routeagent_controller.go.nolint

```diff
@@ -92,19 +94,20 @@ func (r *ReconcileRouteagent) Reconcile(request reconcile.Request) (reconcile.Re
 	}
 
 	// Define a new Pod object
```

Comment:
nit: Obviously comment is obsolete (and either way is pointless)

Reply:
Done